saleor/checkout/migrations/0041_auto_20220329_1612.py

```diff
@@ -0,0 +1,157 @@
+# Generated by Django 3.2.12 on 2022-03-29 16:12
+
+from decimal import Decimal
+
+import django.utils.timezone
+from django.db import migrations, models
+
+
+def set_default_checkout_line_currency(apps, schema_editor):
+    CheckoutLine = apps.get_model("checkout", "CheckoutLine")
+    lines = CheckoutLine.objects.select_related("checkout")
+
+    for line in lines:
+        line.currency = line.checkout.currency
+
+    CheckoutLine.objects.bulk_update(lines, ["currency"])
```

Comment:
Maybe?
```suggestion
    CheckoutLine.objects.update(currency=F("checkout__currency"))
```

Reply:
Also, another alternative would be:
```python3
CheckoutLine.objects.update(
    currency=Subquery(
        Checkout.objects.filter(pk=OuterRef("checkout")).values("currency")[:1]
    )
)
```
or raw SQL operation.
They should be equivalent to update with `F` expression.

Reply:
Solved in 
https://github.com/saleor/saleor/pull/9526/commits/43bcac3eee309d29660cdbea3a0fb7ece2a5fb47